src/it/scala/com/databricks/spark/redshift/RedshiftIntegrationSuite.scala

```diff
@@ -240,6 +240,10 @@ class RedshiftIntegrationSuite
       TestUtils.expectedData)
   }
 
+  test("count() on DataFrame created from a Redshift table") {
+    assert(sqlContext.sql("select * from test_table").count() === TestUtils.expectedData.length)
```

Comment:
This is minor, but even for things like this I'd consider using `checkAnswer` and explicit `COUNT(*)`, just cause when things do go wrong the query plans can be insightful.